### ISSUE TITLE
`extends` operator not handled

### ISSUE THREAD

**Issue by eventualbuddha** [juliankrispel/decaf#7]
```coffee
a extends b
```

Currently becomes `b`, but should probably be something like `Object.assign(a, b);`

**Comment by eventualbuddha** [comment]
Actually, it's more complicated than that. This is what the official CoffeeScript project generates:

```js
var extend = function(child, parent) { for (var key in parent) { if (hasProp.call(parent, key)) child[key] = parent[key]; } function ctor() { this.constructor = child; } ctor.prototype = parent.prototype; child.prototype = new ctor(); child.__super__ = parent.prototype; return child; },
  hasProp = {}.hasOwnProperty;

extend(a, b);
```

This is probably bespoke enough that it should be reproduced as-is.

**Comment by juliankrispel** [comment]
@eventualbuddha big fan of your work dude, thanks for the issues :)

**Comment by eventualbuddha** [comment]
No problem! Glad to see more work done on this problem, even if it's not in my project.

**Comment by carlmathisen** [comment]
You guys should join forces. :-)

**Comment by juliankrispel** [comment]
fixed in #93
